corehq/apps/style/static/style/less/bootstrap3/report.less

```diff
@@ -15,8 +15,8 @@
   .border-top-radius(5px);
   .border-bottom-radius(5px);
   left: 50%;
-  width: 150px;
-  margin-left: -75px;
+  width: 350px;
+  margin-left: -175px;
```

Comment:
just read this little blog and they seem to think they aren't sketch: https://www.smashingmagazine.com/2009/07/the-definitive-guide-to-using-negative-margins/. fine to leave